tests/test_h5store.py

```diff
@@ -201,6 +201,7 @@ def test_repr(self):
         with self.open_h5store() as h5s:
             key = 'test_repr'
             h5s[key] = self.get_testdata()
+            repr(h5s[key])
```

Comment:
In `freud`, I test these `repr`s that correspond to constructor arguments with `assert repr(obj) = repr(eval(repr(obj)))` or something like that. Could be useful as a check.

Reply:
This will only work in combination with `from signac.core.h5store import H5Group` unless I add the whole module path, thoughts?

Reply:
I'll follow the `numpy` example, that does not include the module name for `np.ndarray`.